src/main/java/io/phdata/streamliner/schemadefiner/model/Jdbc.java

```diff
@@ -53,6 +53,10 @@ public class Jdbc extends Source {
    * */
   public Set<String> ignoreTables;
 
+  /* Flag to decide extra attributes(location & fileType) should be crawled from Hive/Impala after schema crawler operation is done.
+   * This can be avoided if location and fileType is not needed for faster operation */
+  public boolean includeHiveAttributes;
```

Comment:
@akuma432 Does this default to false?

Reply:
It shouldn't be required/break existing customers if it's not specified

Reply:
Yes it default's to false. If the user has not specified it in the config, this will be false. It's not mandatory to mention it it the config.